### PR TITLE
Add CONTRIBUTING.md to prepare for move to Google github

### PR DESCRIPTION
**Pull request by bryanoltman**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# How to Contribute
+
+We'd love to accept your patches and contributions to this project.
```
Comment (xster) [inline CONTRIBUTING.md]: I'm holding off a bit on this one. Checking with Hans for capacity on his team. We might have to phrase this more conservatively in the interim.

Comment (bryanoltman) [reply CONTRIBUTING.md]: Sure thing, I just used the provided template for this, so suggestions for changes are welcome.

Comment (jacobsimionato) [reply CONTRIBUTING.md]: This is fine by me, but obviously wait to see what Hans thinks.

**Review by jacobsimionato — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (xster) [reply CONTRIBUTING.md]: I don't have an answer for this yet, so let's just say we're not taking contributions for this project currently.

Comment (bryanoltman) [reply CONTRIBUTING.md]: Done.

**Comment by bryanoltman** [comment]
Fixed by d3d0d72df4a05206a0a156e160a61b5deaee2293

**Pull request closed without merging**